poetry.lock

```diff
@@ -1,21 +1,212 @@
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
I got errors trying to install packages from this, and had to run `poetry lock` to fix/update this file before I could run `poetry install`, but judging by this opening message that's plausibly because we're running on slightly different versions of poetry. All that is to say, worth checking this behaves for you, but if it does I'm happy to just accept that I should strictly speaking match your poetry version so nothing to change!